src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c

```diff
@@ -1350,7 +1350,7 @@ int wm_vuldet_send_agent_report(sqlite3 *db, OSHash *cve_table, agent_software *
     wdb_finalize(stmt);
 
     if (OS_INVALID == cve_insert_result) {
-        mtwarn(WM_VULNDETECTOR_LOGTAG, "Failed to insert vulnerabilities in the agent %.3d database", atoi(agents_it->agent_id));
+        mtdebug1(WM_VULNDETECTOR_LOGTAG, "Failed to insert vulnerabilities in the agent %.3d database", atoi(agents_it->agent_id));
```

Comment:
We can avoid the duplicated entry error by modifying the **WDB_STMT_VULN_CVE_INSERT** statement to do INSERT OR IGNORE like:
`"INSERT INTO vuln_cves (name, version, architecture, cve) VALUES(?,?,?,?);"`
This will let us conserve the warning as a way to know that something unexpected happens.

Reply:
Fixed